io/plugins/eu.esdihumboldt.hale.io.jdbc.test/src/eu/esdihumboldt/hale/io/jdbc/test/DBImageParameters.java

```diff
@@ -0,0 +1,96 @@
+package eu.esdihumboldt.hale.io.jdbc.test;
+
+import java.net.URI;
+
+
+public class DBImageParameters {
```

Comment:
I would much prefer to have an interface instead of this class for this purpose.

For the following reasons:
* instead of just providing configuration data, methods can be used to allow more flexible configuration (e.g. for building the JDBC URL, in some cases the "Start URL" may not be sufficient)
* the configuration object should not be mutable
* if we decide later to do other tests than database tests with Docker, we can extract part of the interface that is relevant for Docker as well a the corresponding test logic and use it for both kind of tests